pkg/apps/util/appserialization/encode.go

```diff
@@ -66,15 +68,15 @@ func MakeDeployment(config *appsv1.DeploymentConfig) (*v1.ReplicationController,
 	for k, v := range config.Spec.Selector {
```

Comment:
if you're trying to make a pure package, `MakeDeployment` doesn't make sense to move.

I still question the value.